temporian/core/serialization.py

```diff
@@ -313,14 +415,20 @@ def get_node(key):
     return op
 
 
-def _serialize_node(src: Node) -> pb.Node:
+def _serialize_node(src: Node, operators: Set[base.Operator]) -> pb.Node:
     assert len(src.schema.features) == len(src.feature_nodes)
+    logging.info("aaaa")
+    logging.info(operators)
     return pb.Node(
         id=_identifier(src),
         sampling_id=_identifier(src.sampling_node),
         feature_ids=[_identifier(f) for f in src.feature_nodes],
         name=src.name,
-        creator_operator_id=_identifier_or_none(src.creator),
+        creator_operator_id=(
+            _identifier_or_none(src.creator)
+            if src.creator is not None and src.creator in operators
+            else None
+        ),

```

Comment:
@achoum had to add these to remove creator operator references from input nodes to the graph. maybe it would make more sense to add it directly in the graph inference code? to discuss tomorrow. related to whether we want to "reset" an eventset's node's previous graph when calling .node()